packages/flutter/lib/src/widgets/form.dart

```diff
@@ -68,34 +74,41 @@ class FormState extends State<Form> {
 
   @override
   Widget build(BuildContext context) {
+    if (config.autovalidate)
+      _validate();
     return new _FormScope(
       formState: this,
       generation: _generation,
       child: config.child
     );
   }
 
-  /// Saves every FormField that is a descendant of this Form.
+  /// Saves every [FormField] that is a descendant of this [Form].
   void save() {
     for (FormFieldState<dynamic> field in _fields)
       field.save();
   }
 
-  /// Resets every FormField that is a descendant of this Form back to its
+  /// Resets every [FormField] that is a descendant of this [Form] back to its
   /// initialState.
   void reset() {
     for (FormFieldState<dynamic> field in _fields)
       field.reset();
     _fieldDidChange();
   }
 
-  /// Returns true if any descendant FormField has an error, false otherwise.
-  bool get hasErrors {
-    for (FormFieldState<dynamic> field in _fields) {
-      if (field.hasError)
-        return true;
-    }
-    return false;
+  /// Validates every [FormField] that is a descendant of this [Form], and
+  /// returns true iff there are no errors.
```

Comment:
"iff" means [if and only if](https://en.wikipedia.org/wiki/If_and_only_if), but if that's not widely known I can change it.